app-core/src/main/java/net/spals/appbuilder/app/core/generic/GenericWorkerApp.java

```diff
@@ -34,13 +39,21 @@
 
     public static class Builder extends GenericWorkerApp_Builder implements WorkerAppBuilder<GenericWorkerApp> {
 
+        private final List<Module> orderedModules = new ArrayList<Module>();
         private final LifecycleInjectorBuilder lifecycleInjectorBuilder;
         private final ServiceGraph serviceGraph = new ServiceGraph();
 
         private final AutoBindConfigModule.Builder configModuleBuilder;
         private final AutoBindServiceGraphModule.Builder serviceGraphModuleBuilder;
+
         private final AutoBindServicesModule.Builder servicesModuleBuilder =
                 new AutoBindServicesModule.Builder();
+        // Track the index of the services module so that it is added to
```

Comment:
No, this is part of the hand written code that working towards creating an `Injector` instance to set in the builder.